vars/execIQETests.groovy

```diff
@@ -74,6 +74,15 @@ def call(args = [:]) {
         ]
     )
 
+    // Add text field for test image
+    p.add(
+        [
+            $class: 'StringParameterDefinition',
+            name: "image", defaultValue: defaultImage ? defaultImage : "",
```

Comment:
Where is `defaultImage` defined? You could use `pipelineVars.iqeCoreImage` -- or were you intending to pass it into this function somewhere?

Reply:
Maybe you intended to set it up here... https://github.com/RedHatInsights/insights-pipeline-lib/pull/112/files#diff-1a8e6545adb1b39aba248f464ddf69b8840cee1e7f663b2896ec87f68ce88342R26

Reply:
Ah good catch, I was following the format used for the `marker` and `filter`. So it will pass an empty string if not populated and then if we have an empty string for `iqeCoreImage` we use https://github.com/RedHatInsights/insights-pipeline-lib/pull/112/files#diff-1a8e6545adb1b39aba248f464ddf69b8840cee1e7f663b2896ec87f68ce88342R26

So if this approach is OK, I just need to add a `defaultImage = args.get('defaultImage')` like https://github.com/RedHatInsights/insights-pipeline-lib/blob/master/vars/execIQETests.groovy#L26